.env.example

```diff
@@ -1,2 +1,2 @@
 # Do not share your OpenAI API key with anyone! It should remain a secret.
-OPENAI_API_KEY=
+OPENAI_API_KEY= "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"
```

Comment:
I think you pushed your API Key by mistake.
Please remove it.

Reply:
You should not expose api key publicly, Check the comment of the file.